packages/cli/package.json

```diff
@@ -80,6 +81,7 @@
     "@preact/async-loader": "^3.0.1",
     "@prefresh/babel-plugin": "^0.4.1",
     "@prefresh/webpack": "^3.2.2",
+    "@types/webpack": "^4.38.0",
```

Comment:
Needed to provide types for `helpers.webpack`. Might be overkill though? Could just `Record<string, unknown>` it.

Reply:
Looking at the lock file I think it's fine to add